modules/user/user.service.js

```diff
@@ -180,6 +182,63 @@ class UserService {
     };
   }
 
+  async googleUser(id_token) {
+    const client = new OAuth2Client();
+    const ticket = await client.verifyIdToken({
+      idToken: id_token,
+      audience: process.env.REACT_APP_GOOGLE_CLIENT_ID,
+    });
+    const payload = ticket.getPayload();
```

Comment:
Rename this variable to something like `ticketPayload`

Reply:
Fixed